app/src/main/res/drawable/ic_visibility_off.xml

```diff
@@ -0,0 +1,5 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android" android:height="24dp" android:tint="#000000" android:viewportHeight="24" android:viewportWidth="24" android:width="24dp">
```

Comment:
Please make sure both icons are formatted the same as all existing icons, including the same attributes for tint.